docs/semgrep-secrets/generic-secrets.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: generic-secrets
+title: Scan for generic secrets
+hide_title: true
+description: Use Semgrep to identify generic secrets in your code.
+tags:
+ - Semgrep Assistant
+ - Semgrep Secrets
+---
+
+import PL from '@site/src/components/Placeholder';
+
+# Generic secrets (beta)
+
+:::info
+Generic secrets is in beta. Please contact [Support](/support) to request access to this feature.
+:::
+
+Generic secrets detection scans your code for the inadvertent inclusion of credentials, such as API keys, passwords, and access tokens using rules. It then analyzes the results to eliminate false positives, so you only see high-signal results likely to be true positives.
+
+## Prerequisites
+
+To scan your code for generic secrets, you must have the following:
+
+- [Repositories hosted by GitHub.com onboarded to Semgrep using Semgrep Managed Scans](/deployment/managed-scanning/github)
```

Comment:
Does this mean we only support generic secrets for GitHub.com?

Reply:
@LewisArdern On the Notion doc, I thought I saw this is for GitHub.com repos only, but it's no longer there. Has this changed?